xen/arch/arm/coproc/coproc.c

```diff
@@ -587,7 +597,10 @@ void __init coproc_init(void)
     unsigned int num_coprocs = 0;
     int ret;
 
-    register_keyhandler('c', coproc_debug_toggle, "toggle debug for coproc", 0);
+    register_keyhandler('c', coproc_debug_toggle,
+                        "increase debug level for coproc", 0);
```

Comment:
While debugging and because we use serial console (115200) for logs print outs it is sometimes not possible to cycle from _VERBOSE back to _ERROR because of huge number of prints possible.
Thus, I sometimes cannot even get back from _VERBOSE to _DEBUG, hence 2 key handlers.
Probably, when we stabilize the framework we can think of a better approach